### PR TITLE
Fixed the index.ipynb Hyperlinks

### LINKED ISSUE #1062: Hyperlinks in index Jupyter Notebook are not working
Hello, 
I was going through the tutorial notebook present [here](https://mybinder.org/v2/gh/poliastro/poliastro/master?filepath=index.ipynb.). But to my dismay, whichever link I clicked on was showing a **404 Error could not find the requested page.** 
Please do confirm the same & Hence, we can work on it to fix this unexpected behaviour :smile:

### PR DESCRIPTION
**Pull request by DhruvSondhi**
Hello,
This PR aims to fix #1062. Have fixed all the Hyperlinks with the correct file & directory. Cheers :)

### CODE REVIEW

**Comment by astrojuanlu** [comment]
Merged! Thanks!

File: index.ipynb [hunk 1]
```diff
@@ -12,65 +12,65 @@
...
+    "[![mars](docs/source/examples/jupiter.png)](docs/source/examples/Going%20to%20Jupiter%20with%20Python%20using%20Jupyter%20and%20poliastro.md)\n",
     "\n",
-    "## [Going to Mars with Python using poliastro](docs/source/examples/Going%20to%20Mars%20with%20Python%20using%20poliastro.ipynb)\n",
+    "## [Going to Mars with Python using poliastro](docs/source/examples/Going%20to%20Mars%20with%20Python%20using%20poliastro.md)\n",
     "\n",
-    "[![mars](docs/source/examples/trans_30_260.png)](docs/source/examples/Going%20to%20Mars%20with%20Python%20using%20poliastro.ipynb)\n",
+    "[![mars](docs/source/examples/trans_30_260.png)](docs/source/examples/Going%20to%20Mars%20with%20Python%20using%20poliastro.md)\n",
     "\n",
-    "## [Propagation using Cowell's formulation](docs/source/examples/Propagation%20using%20Cowell's%20formulation.ipynb)\n",
+    "## [Propagation using Cowell's formulation](docs/source/examples/Propagation%20using%20Cowell's%20formulation.md)\n",
```
Comment (jorgepiloto) [inline index.ipynb:32]: This must be:

```markdown
docs/source/examples/Propagation%20using%20Cowells%20formulation.md
```

File: index.ipynb [hunk 2]
```diff
@@ -12,65 +12,65 @@
...
     "\n",
-    "[![cowell](docs/source/examples/cowell.png)](docs/source/examples/Propagation%20using%20Cowell's%20formulation.ipynb)\n",
+    "[![cowell](docs/source/examples/cowell.png)](docs/source/examples/Propagation%20using%20Cowell's%20formulation.md)\n",
     "\n",
-    "## [Revisiting Lambert's problem in Python](docs/source/examples/Revisiting%20Lambert's%20problem%20in%20Python.ipynb)\n",
+    "## [Revisiting Lambert's problem in Python](docs/source/examples/Revisiting%20Lambert's%20problem%20in%20Python.md)\n",
```
Comment (jorgepiloto) [inline index.ipynb:36]: This must be:

```markdown
docs/source/examples/Revisiting%20Lamberts%20problem%20in%20Python.md
```

**Review by jorgepiloto — COMMENTED** [review]
No check failures were raised because this file is not being tested anywhere.

File: index.ipynb (hunk 1 above)
Comment (DhruvSondhi) [reply index.ipynb:32]: Sure thing, Understood. But why was it working locally then?

File: index.ipynb (hunk 2 above)
Comment (DhruvSondhi) [reply index.ipynb:36]: Sure thing, fixing this in the next commit

**Comment by astrojuanlu** [comment]
Well spotted, thanks @jorgepiloto !

**Comment by DhruvSondhi** [comment]
I have pushed the commits with the requested [change ](https://github.com/poliastro/poliastro/pull/1067/commits/66b3ac84fb844f577ac590fab4e8375cbb7c076b)

**Pull request merged**